locales-en-US.xml

```diff
@@ -313,6 +550,19 @@
     <term name="editortranslator" form="verb">edited &amp; translated by</term>
 
     <!-- SHORT VERB ROLE FORMS -->
+    <term name="chair" form="verb-short"></term> <!-- ??? -->
+    <term name="compiler" form="verb-short">compiled by</term>
```

Comment:
I don't think we'll find an authoritative source here. I don't think any of these are in MLA or Chicago and thus likely also missing from other works. We could just take best guesses? (e.g. "comp. by" is likely right, and I'd abbreviate with as "w.", etc.) Any other ideas?

Reply:
What do you think of the below?